internal/provider/incident_catalog_entries_resource.go

```diff
@@ -600,3 +622,58 @@ func (r *IncidentCatalogEntriesResource) reconcile(ctx context.Context, data *In
 
 	return catalogType, entries, nil
 }
+
+// isAttributeManaged checks if the given attribute should be managed by this resource.
+func (m *IncidentCatalogEntriesResourceModel) isAttributeManaged(attributeID string) bool {
+	// Check if the attribute is in the managed list
+	attrSet, known := m.managedAttributesSet()
+	if !known {
+		return true
+	}
+
+	return attrSet[attributeID]
+}
+
+func (m *IncidentCatalogEntriesResourceModel) managedAttributesSet() (map[string]bool, bool) {
+	if m.ManagedAttributes.IsNull() || m.ManagedAttributes.IsUnknown() {
+		return nil, false
+	}
+
+	if m.managedAttrSet != nil {
```

Comment:
I think we should use len(m.managedAttrSet) == 0 here. If someone sets it empty?

Reply:
imo if someone sets it to `[]` that means "no attributes are managed" vs not setting it which means "all attributes are managed".

It wouldn't be _totally_ wild to only manage the `name` / `external_id` / `aliases` in tf and have a bunch of dashboard-managed attributes imo!